syntax = "proto3";
package agent;





/*
ListAgentStatusOld请求
*/
message ListAgentStatusOldRequest {
    /*
    agentIP
    */
    string ip = 1 ;
    /*
    agentIP列表，逗号分割
    */
    string ip__in = 2 ;
}

/*
ListAgentStatusOld返回
*/
message ListAgentStatusOldResponse {
message List {
message Status {
    /*
    连接开始时间戳
    */
    int32 connect_time = 1 ;
    /*
    agent端心跳时间戳
    */
    int32 hb_agent_time = 2 ;
    /*
    server端心跳时间戳
    */
    int32 hb_server_time = 3 ;
    /*
    agent启动时间戳
    */
    int32 start_time = 4 ;
    /*
    agent版本
    */
    string version = 5 ;
    /*
    gateway信息
    */
    string gateway = 6 ;
}
        
    
    /*
    状态信息列表
    */
    repeated Status status = 1 ;
    /*
    是否存活
    */
    bool alive = 2 ;
    /*
    是否存活
    */
    string version = 3 ;
    /*
    UUID
    */
    string uuid = 4 ;
    /*
    时区
    */
    int32 timezone = 5 ;
    /*
    agent端心跳时间戳
    */
    int32 hb_agent_time = 6 ;
    /*
    server端心跳时间戳
    */
    int32 hb_server_time = 7 ;
    /*
    agent启动时间戳
    */
    int32 start_time = 8 ;
    /*
    agent
    */
    string agent = 9 ;
}
        
    
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated List list = 4 ;
}

/*
ListAgentStatusOldApi返回
*/
message ListAgentStatusOldResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    agent.ListAgentStatusOldResponse data = 4 ;
}

